scripts/generator/templates/template-release-notes.md

```diff
@@ -1,187 +1,132 @@
-# Vaadin {{platform}}
+[Changelogs](#_changelogs) · [Upgrading guides](#_upgrading_guides) · [Docs](https://vaadin.com/docs/latest/)
 
-Vaadin consists of a set of web components, a Java web framework, configurable themes, tools and a set of app templates.
+{{changesSincePrevious}}
 
-Visit [vaadin.com](https://vaadin.com/) to get started.
+## <a id="_changelogs"></a> Changelogs
 
-## New and Noteworthy Since vaadin 22
+- Flow and Fusion ([{{core.flow.javaVersion}}](https://github.com/vaadin/flow/releases/tag/{{core.flow.javaVersion}}))
+- Design System
+  - Web Components ([{{core.vaadin-accordion.jsVersion}}](https://github.com/vaadin/web-components/releases/tag/v{{core.vaadin-accordion.jsVersion}}))
+  - Flow Components ([{{platform}}](https://github.com/vaadin/flow-components/releases/tag/{{platform}}))
+- Collaboration Engine ([{{vaadin.vaadin-collaboration-engine.javaVersion}}](https://github.com/vaadin/collaboration-engine/releases/tag/{{vaadin.vaadin-collaboration-engine.javaVersion}}))
+- Designer ([Release notes](https://github.com/vaadin/designer/blob/master/RELEASE-NOTES.md))
+- TestBench ([{{vaadin.vaadin-testbench.javaVersion}}](https://github.com/vaadin/testbench/releases/tag/{{vaadin.vaadin-testbench.javaVersion}}))
+- Multiplatform Runtime (MPR) ([{{core.mpr-v8.javaVersion}}](https://github.com/vaadin/multiplatform-runtime/releases/tag/{{core.mpr-v8.javaVersion}}))
+- Router ([{{core.vaadin-router.jsVersion}}](https://github.com/vaadin/vaadin-router/releases/tag/v{{core.vaadin-router.jsVersion}}))
 
-Here are the highlighted new and improved features in vaadin 23. To see the full list of bug fixes and improvements, check Included Projects and Change Log.
+**Official add-ons and plugins:**
+
+- Spring add-on ([{{core.flow-spring.javaVersion}}](https://github.com/vaadin/spring/releases/tag/{{core.flow-spring.javaVersion}}))
+- CDI add-on ([{{core.flow-cdi.javaVersion}}](https://github.com/vaadin/cdi/releases/tag/{{core.flow-cdi.javaVersion}}))
+- Maven plugin ({{platform}})
+- Gradle plugin ({{platform}})
+- OSGi plugin ([{{core.flow-osgi.javaVersion}}](https://github.com/vaadin/osgi/releases/tag/{{core.flow-osqi.javaVersion}}))
+- Quarkus plugin ([{{core.vaadin-quarkus.javaVersion}}](https://github.com/vaadin/quarkus/releases/tag/{{core.vaadin-quarkus.javaVersion}}))
+
+
+## <a id="_upgrading_guides"></a> Upgrading guides
+
+- [Upgrading Flow to Vaadin 23](https://vaadin.com/docs/latest/flow/upgrading/changes/#changes-in-vaadin-23)
+- [Upgrading Fusion to Vaadin 23](https://vaadin.com/docs/latest/fusion/upgrading/changes/#changes-in-vaadin-22)
```

Comment:
>[Upgrading Fusion to Vaadin 23](https://vaadin.com/docs/latest/fusion/upgrading/changes/#changes-in-vaadin-22)

this link refers to changes in vaadin 22. i think it should be vaadin 23, right?

Reply:
Yes, thanks, I apparently missed that.

Reply:
```suggestion
- [Upgrading Fusion to Vaadin 23](https://vaadin.com/docs/latest/fusion/upgrading/changes/#changes-in-vaadin-23)
```